posts/rirods-0-2-0/index.qmd

```diff
@@ -18,13 +18,36 @@ here::i_am("posts/rirods-0-2-0/index.qmd")
 ```
 
 ::: {.callout-note}
-This article is about  release 0.2.0 of `rirods` (see the [documentation](http://rirods.irods4r.org)).
+This article is about the release of `rirods` 0.2.0 (see the [documentation](http://rirods.irods4r.org)). iRODS C++ HTTP 0.2.0 is a requirement for this R package.
 :::
 
+As of November 2023 [iRODS](https://irods.org) has released the [iRODS C++ HTTP API](https://irods.org/2023/11/initial-release-of-the-irods-http-api/), as a replacement of the old [iRODS C++ REST API](https://github.com/irods/irods_client_rest_cpp). This new interface to the iRODS server comes with a host of improvements including flexibility of management for all types of operations on users, collections and data objects, as well as supporting authentication with [OpenID Connect](https://openid.net/developers/how-connect-works/). The HTTP API is steadily maturing, so it was about time to leverage this functionality with `rirods`.
 
-Packages used in this blog post.
+Besides a lot of potential for future rirods development in terms of iRODS operations, this new release (rirods 0.2.0) ensures some obvious improvements on the same core functionality as in previous releases of rirods. In this blog we will showcase one of those improvements---the speed of uploading and downloading of files to iRODS. The new HTTP API can now be configured to allow receiving of much larger payloads. An example of how the HTTP API can be configured with a JSON file can be seen [here](https://github.com/irods/irods_demo/blob/main/irods_client_http_api/config.json). On top of that, write requests can now be made in parallel, which can as well be configured for the number of allowed threads.
 
-```{r echo=TRUE, message=FALSE}
+In order to benchmark this improvement we show upload and download time as well as memory usage for `iget()` and `iput()`, respectively. We use the Docker based [iRODS demo](https://github.com/irods/irods_demo) as our server. We can easily stand this server up, as follows:
+
+```{r echo=TRUE}
+library(rirods)
+use_irods_demo()
+```
+
+For more information on `use_irods_demo()` see the `demo` vignette (`vignette("demo", package = "rirods")`) or this [page](https://rirods.irods4r.org/articles/demo.html). Now that the demo server is running we can log in to the system by following the instructions as printed above.
```

Comment:
The linked demo.html page still mentions REST API with links/examples.   Should updating that page be part of this same PR?

Reply:
That is part of the rirods PR